### PR TITLE
Add basic test suite

### PR DESCRIPTION
**Pull request by maxmeyer**
I added a basic test suite and fixed a long that some kind of things.

At first I could not load the library correctly in tests. Therefore I make those requires explicit and place everything at `geo_pattern.rb` to make it available at once.

This PR is based on the rspec-PR so please merge in that order.

Cheers.

### CODE REVIEW

File: lib/geo_pattern/pattern_helpers.rb [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Dir[File.join(File.dirname(__FILE__), '**', '*.rb')].each { |file| require file }
```
Comment (ttaylorr) [inline lib/geo_pattern/pattern_helpers.rb]: Why make all of the `require`s explicit instead of using this?

**Comment by ttaylorr** [comment]
Looks alright to me, but some of the testing behavior that we're introducing worries me a bit.  I think once my comments are addressed we'll be good to merge, but I want to look further into a real implementation of `SVG`, instead of just wrapping a string.

**Comment by jasonlong** [comment]
Thanks for the help here @ttaylorr, this is great. And thanks @maxmeyer for continuing to get this polished up.

Are we good to merge here or are there some outstanding questions yet?

**Comment by ttaylorr** [comment]
Always glad to help -- this PR looks ready to ship from my perspective. :)






On Thursday, Jan 8, 2015 at 1:13 PM, Jason Long <notifications@github.com>, wrote:

Thanks for the help here @ttaylorr, this is great. And thanks @maxmeyer for continuing to get this polished up.


Are we good to merge here or are there some outstanding questions yet?

**Comment by jasonlong** [comment]
:boom: Boom, 1 down!

**Comment by maxmeyer** [comment]
Thanks. :+1:

**Pull request merged**